### ISSUE TITLE
Kuberenetes References Missing In jballerina-1.1.2

### ISSUE THREAD

**Issue by jimk-osu** [ballerina-platform/ballerina-lang#21141]
**Description:**
jballerina-1.1.2 is missing references/jars for Kubernetes.

Have tried using both `ballerina\kuberenetes` and `ballerinax\kuberenetes` each causing the following error during build time:

```shell
cannot resolve module 'ballerinax/kubernetes'
```

I have also checked and there are differences in the `./distribution/bre/lib` between 1.1.1 and 1.1.2:

1.1.1:
```
pwd
/Library/Ballerina/distributions/jballerina-1.1.1/bre/lib
ls | grep kubernetes
ballerina-kubernetes-1.1.1.jar
kubernetes-extension-1.1.1.jar
kubernetes.jar
```

1.1.2:
```
pwd
/Library/Ballerina/distributions/jballerina-1.1.2/bre/lib
ls | grep kubernetes
kubernetes-extension-1.1.2.jar
```

Going further, nothing is being returned in central.ballerina.io for kubernetes either.

**Steps to reproduce:**
- use jballerina-1.1.2
- generate project and use kubernetes annotations + import
- build

**Affected Versions:**
`jballerina-1.1.2`

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by anuruddhal** [comment]
Thanks for reporting the issue. 
The jars are missing in the pack download from the `ballerina dist update` command. 
However,  Ballerina 1.1.2 pack installed from the installers contains the jars and annotations are working.

We will do a new release (1.1.3) today fixing this issue.

**Comment by hemikak** [comment]
Hi @jimk-osu, We have released 1.1.3 version with the mentioned issue fixed.
